CONTRIBUTING.md

```diff
@@ -89,16 +89,14 @@ There are examples of such models in sa/model.go, along with code to
 turn a model into a `struct` used internally. 
 
 An example of a flag-gated migration, adding a new `IsWizard` field to Person
-controlled by a `allowWizards` flag:
+controlled by a `AllowWizards` feature flag:
```

Comment:
To keep this example self-contained could you add the `features/features.go` diff required to add the `AllowWizards` feature?